docs(themeing): Move all themes in separate files (#1782)
IgniteUI/igniteui-docfx: en/components/themes/global-theme.md

#### igx-theme  

| Name              |  Type   | Default            | Description                                                                                                    |
| :---------------: | :-----: | :---------------:  | :------------------------------------------------------------------------------------------------------------: |
| `$palette`        | map     | undefined          | The palette map to be used to by the default themes of all components.                                         |
| `$schema`         | map     | $light-schema      | The schema used as basis for styling the components.                                                           |
| `$exclude`        | list    | ( )                | A list of component themes to be excluded from the global theme.                                               |
| `$legacy-support` | boolean | `true`             | Determines the theming strategy - if set to false, theming is done via CSS variables.                          |
| `$roundness`      | Number  |  null              | Sets the global roundness factor (the value can be any decimal fraction between 0 and 1) for all components..  |

> | `$elevation`      | boolean | `true`             | Determines the theming strategy - if set to false, theming is done via CSS variables.                          |

Reviewer: The wording here is wrong.

```
Ideally you would be applying `.light-theme` and `.dark-theme` CSS classes somewhere high in your application DOM tree. Your `app-root` element is a good candidate for that.

#### Available Themes

> IgniteUI for angular gives you the option to pick from a set of predefined themes.  

Reviewer: ```suggestion
Ignite UI for Angular gives you the option to pick from a set of predefined themes.
```

| **Material (light)**|  igx-light-theme()           | `$light-schema          ` | $default-palette                                        |      **6.2 +**       |
| **Material (dark)** |  igx-dark-theme()            | `$dark-schema           ` | $dark-palette                                           |      **6.2 +**       |
| **Fluent**          |  igx-fluent-theme()          | `$light-fluent-schema   ` | $fluent-excel-palette <br> $fluent-word-palette           |      **8.2 +**       |
| **Fluent**          |  igx-fluent-theme()          | `$light-fluent-schema   ` | $fluent-excel-palette <br> $fluent-word-palette           |      **8.2 +**       |
| **Fluent (dark)**   |  igx-fluent-dark-theme()     | `$dark-fluent-schema    ` | $fluent-excel-dark-palette or $fluent-word-dark-palette |      **8.2 +**       |
| **Bootstrap**       |  igx-bootstrap-theme()       | `$light-bootstrap-schema` | $bootstrap-palette                                      |      **9.0 +**       |
| **Bootstrap (dark)**|  igx-bootstrap-dark-theme()  | `$dark-bootstrap-schema ` | $bootstrap-dark-palette                                 |      **9.0 +**       |

> > [!IMPORTANT]

Reviewer: This should rather be a note. Not an important section.

> > keep in mind that all theme extend the `base` theme `igx-theme`.

Reviewer: ```suggestion
> Note that all high-level theme mixins wrap the base `igx-theme` mixin.
```

> All themes can be used as a starting point to create your own theme. For example lets start our theme by using `igx-bootstrap-theme` as a starting point.

Reviewer: ```suggestion
All theme mixins can be used as a starting point to create your own theme. Let's create a new theme by using the `igx-bootstrap-theme` mixin.
```
